binding/Binding/SKPngChunkReader.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace SkiaSharp
+{
+	/// <summary>
+	/// Base class for optional callbacks to retrieve meta/chunk data out of a PNG
+	/// <br></br> encoded image as it is being decoded.
+	/// <br></br> Used by SkCodec.
+	/// </summary>
```

Comment:
All docs unfortunately need to be done in the separate repo: https://github.com/mono/SkiaSharp-API-docs

However, this repo is only updated AFTER we merge the PR here... so yeah,  🐔 ♻️🐣 ♻️ 🍗 ♻️ 🍳